taxi/views.py

```diff
@@ -87,3 +119,35 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+    @staticmethod
+    def post(request, *args, **kwargs):
+        driver = get_object_or_404(Driver, pk=kwargs["pk"])
+        form = DriverLicenseUpdateForm(request.POST)
+
+        if form.is_valid():
+            driver.license_number = form["license_number"].data
+            driver.save()
```

Comment:
You can call form.save()

Reply:
Done. How it work? Because I change driver instance, why should I save form?

Reply:
Actually, it does not work. License does not change after updating.

Reply:
I got an error. Send below